packages/test-runner/test/assets/parametrized/test.spec.ts

```diff
@@ -15,8 +15,12 @@
  */
 
 import { fixtures as baseFixtures } from '../../..';
+import { registerParameter } from '../../../out/fixtures';
```

Comment:
Why do we import from 'out', and not use fixtures.registerParameter?

Reply:
i guess it is autocomplete